src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -40,6 +45,125 @@
         }
     ],
     "metrics": [
+        {
```

Comment:
will revert vscodeTelemetry.json once common definition is updated

Reply:
Do we want to do the common repo update first and then update the lib dep version?

Reply:
>once common definition is updated
If this is ongoing then it's good.

Reply:
yep will update common repo first then bump the telemetry version here instead of overwriting vscode.telemtry